signac_dashboard/dashboard.py

```diff
@@ -448,10 +452,11 @@ class MyDashboard(Dashboard):
         """
         if import_file is not None:
             import_name = import_file + "." + import_name
+        view = LazyView(dashboard=self, import_name=import_name)
```

Comment:
This change is what enables the same endpoint to go to multiple urls. See https://flask.palletsprojects.com/en/2.0.x/patterns/lazyloading/?highlight=endpoint